### ISSUE TITLE
batchnorm training mode in convnet_maml

### ISSUE THREAD

**Issue by buptxiaofeng** [Han-Jia/UNICORN-MAML#8]
https://github.com/Han-Jia/UNICORN-MAML/blob/c66b9f347f52afb5a988e4c679c9d398a41f9ae3/model/networks/convnet_maml.py#L31

Hi, thanks again for this impressive work. I really learned a lot. I have a question about the batchnorm in the implementation of convnet_maml. It seems that the variable self.training is always True. In the implementation of nn.BatchNorm2d, this variable is controlled by :
  if self.training:
            bn_training = True
  else:
            bn_training = (self.running_mean is None) and (self.running_var is None)

then F.batch_norm is applied.
https://pytorch.org/docs/stable/_modules/torch/nn/modules/batchnorm.html#BatchNorm2d

Since in the forward of convnet_maml, F.batch_norm is applied, and the self.training is alway True, the batch_norm is always in the training mode, isn't it?
